user-manual/import-export/upload-digital-object.rst

```diff
@@ -191,6 +193,186 @@ your :term:`digital object`.
+      reviewed for personal or confidential information. Please contact the 
+      Reference Archivist to request access and initiate an access review.".
+
+:ref:`Back to top <upload-digital-object>`
+
+.. _digital-object-preservation:
+
+Digital object metadata for preservation files
+==============================================
+
+If you or another user of a preservation system (e.g. Archivematica) has sent a DIP upload to an archival description :term:`slug` in AtoM then thumbnails for the digital objects in that Dissemenation Information Package (DIP) will be displayed in the archival description's digital object carousel.
+
+   .. NOTE::
+
+      An unauthenticated user will not see a digital object and its metadata if the archival description it is attached to has a "draft" instead of "published" status.
+
+When selecting one of the digital objects included in the DIP upload, the "Digital object metadata" section on the archival description view page will include a "Preservation Copies" and "Access Copies" subsection.
+
+   .. NOTE::
+
+      When the preservation system uploads a DIP with digital objects to AtoM, it will generate "Master file", "Reference copy", and "Thumbnail copy" representations with their metadata displayed in the "Access copies" subsection.
+
+1. Clicking on either of these subsection label links will collapse or expand 
+   the subsection. See the :ref:`digital-object-metadata` above for more about
+   the "Access copies" section.
+
+2. The "Preservation Copies" subsection will include metadata about the
+   "Original file" and "Preservation copy" representations that are stored in 
+   the preservation system. 
+
+   .. NOTE::
+
+      If the preservation system did not generate a preservation copy then 
+      only the "Original file" section is shown.
+
+3. The "Preservation copy" metadata includes filename, filesize, and a 
+   timestamp for when the preservation system normalized (i.e. transcoded) the 
+   preservation copy from the original file.  
+
+4. The "Original file" metadata may include filename, file format name, file 
+   format version, format registry key (the unique key assigned to the file 
+   format by a format registry), format registry name (e.g. PRONOM), filesize, 
+   and ingested (the timestamp for when the preservation system received the 
+   original file).
+
+5. If the preservation action system statement option is enabled, you will
+   also see a "Permissions" field with custom text that is populated by an Administrator user (e.g. "Access only provided on research room computer.").
+   See **Admin > Settings > Permissions > Preservation system access statement**
+
+   .. image:: images/digital-object-preservation-copies-no-access.*
+      :align: center
+      :width: 90%
+      :alt: digital object preservation copy display, without access
+
+6. An administrator user can toggle the display of specific "Digital object 
```

Comment:
May want to actually add a link to the Visible elements documentation here? Same with above (Settings > Permissions > Preservation system access statement), though not sure if you've written those docs yet?

Reply:
Link added. Also added docs for Preservation system access statement. Had forgotten those in original commit.

Reply:
FYI. Since the Preservation action system access statement is not linked to actionable PREMIS rights functionality, like the rest of the settings in the Permissions settings. I added the access statement docs at the bottom of the Permissions settings section, rather than linking out to the Rights page as is done for the other settings in this section.

Reply:
Sounds good - will keep that in mind when I look at the updates